src/jade/tabs/widgets/balances.jade

```diff
@@ -83,4 +83,4 @@
                 ) {{ component | rpamount:{tiny_precision: 3} }}
 
     p(ng-show="$route.current.tabName == 'balance'")
-      a.btn.btn-cancel(href="#/trust", l10n) add gateway
+      a.btn.btn-cancel(href="#/xrp", l10n) add gateway
```

Comment:
@yongsoo I agree that it should link to the trust page. What do you think @annatonger?